### ISSUE TITLE
update typeing-extensions > 4.1 for python 3.10

### ISSUE THREAD

**Issue by TKaluza** [zauberzeug/nicegui#78]
Using python 3.10.6 with pydantic I got the following conflict:
```
  nicegui 0.8.9 depends on typing-extensions<4.0.0 and >=3.10.0
  pydantic 1.10.1 depends on typing-extensions>=4.1.0
```

https://github.com/zauberzeug/nicegui/blob/9875115bbbc35b2527f9afc007121117d3d6d5a2/pyproject.toml#L15

As far as I can assume I suspect it should be possible to change the requirements or why is <4.0.0 a necessary? 

Thank you very much! :smile_cat:

**Comment by TKaluza** [comment]
I did a short test In a local install I've forced typing-extensions > 4.1 and did  run in any problems. But of course this is not a valid test.

**Comment by rodja** [comment]
Thanks for identifying this issue. I just fixed the dependencies and released [0.8.11](https://github.com/zauberzeug/nicegui/releases/tag/v0.8.11). Happy coding!
